samples/system-test/datasets.test.js

```diff
@@ -27,42 +27,37 @@ const datasetId = `nodejs-docs-samples-test-${uuid.v4()}`.replace(/-/gi, '_');
 const projectId = process.env.GCLOUD_PROJECT;
 const bigquery = new BigQuery();
 
-test.before(tools.checkCredentials);
-test.beforeEach(tools.stubConsole);
```

Comment:
what happened to the beforeEach?

Reply:
Doesn't seems "stubConsole" is required here